typescript/typescript-introduction/tsc-intro/why-typescript.md

```diff
@@ -46,15 +45,14 @@ This is useful because making sure that the same data shapes are used throughout
 
 One way to think of TypeScript is as a **spellchecker for JavaScript**.
 
-TypeScript checks your code **statically**, by looking at it *without running it*. This can speed up development because you can catch problems early on[1].
+TypeScript checks your code **statically**, by looking at it _without running it_. This can speed up development because you can catch problems early on[1].
```

Comment:
```suggestion
TypeScript checks your code **statically**, by looking at it *without running it*. This can speed up development because you can catch problems early on[1].
```